src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,32 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final String CIRCLE_COLOR = "WHITE";
+    private static final int CIRCLE_RADIUS = 10;
+    private static final Random random = new Random();
+    private static final ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Figure getRandomFigure() {
+        String color = colorSupplier.getRandomColor();
+
+        switch (random.nextInt(5)) {
```

Comment:
Magic numbers should be avoided. Please define a constant for the number 5, which represents the number of figure types.